### PR TITLE
test: create verify-changelog.yml

### PR DESCRIPTION
**Pull request by Valastiri**
## Description

Adds a new test to ensure changelog is updated when PRs are ready for review.

Features:
- When action fails:
    -  It will add a comment in the PR: `Please add changelog entry. ❌`.
	- To prevent comment spam the action will check for the above comment before trying to add it to the PR.
- On success:
	- If the changelog was edited prior to the PR being submitted the action will not comment on the PR and simply pass the check.
	- If the action had previously failed and was re-run it will check if `Please add changelog entry. ❌` exists and update it.

Additional: If for whatever reason a changelog entry is not required there is a use case to support this (and is currently built-in via a label) - **requires label creation**

## Screenshots

PR has no changelog modifications, fails check, action will look for comment (does not exist), creates comment:
image

Handling adding extra commits while not modifying changelog + comment already exists:
image

Sample bot message:
image

Replaced by bot upon success:
image

## Discord Username

Valastiri#8902

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Update <small>_ May 2022</small>
 - fix: reword .xbox command and change category (05/05/2022)
 - fix: Include stable in wx command (04/05/2022)
 - chore: add dependabot config (02/05/2022)
+- test: create changelog verification action (6/28/2022)
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:13]: Check the month that this is under :) May want to rebase

**Comment by NathanInnes** [comment]
The workflows need to be split so the jobs have read/write access at different times to prevent the following attack. 
https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

**Comment by Valastiri** [comment]
Changed the way the action works but requires additional work by utilizing `workflow_run`. Setting to Draft until completed.

**Comment by benw202** [comment]
Hey @Valastiri, is this ready to go? Or more work needed?

**Comment by NathanInnes** [comment]
Closing as this was introduced with PR #320

**Pull request closed without merging**